Index: src/main/java/com/example/phoenix/HelloController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.phoenix;\r\n\r\nimport javafx.fxml.FXML;\r\nimport javafx.scene.control.Label;\r\n\r\npublic class HelloController {\r\n    @FXML\r\n    private Label welcomeText;\r\n\r\n    @FXML\r\n    protected void onHelloButtonClick() {\r\n        welcomeText.setText(\"Welcome to JavaFX Application!\");\r\n    }\r\n}
===================================================================
diff --git a/src/main/java/com/example/phoenix/HelloController.java b/src/main/java/com/example/phoenix/HelloController.java
--- a/src/main/java/com/example/phoenix/HelloController.java	
+++ b/src/main/java/com/example/phoenix/HelloController.java	
@@ -9,6 +9,6 @@
 
     @FXML
     protected void onHelloButtonClick() {
-        welcomeText.setText("Welcome to JavaFX Application!");
+        welcomeText.setText("VAFHANCULU!");
     }
 }
\ No newline at end of file
